ci/jenkins/test.sh

```diff
@@ -468,10 +462,59 @@ function deliver_antrea_windows {
         done
     done
     rm -f antrea-windows.tar.gz
+    echo "==== Finish building and delivering Windows Docker images ===="
 }
 
-function deliver_antrea_windows_containerd {
-    echo "====== Cleanup Antrea Installation Before Delivering Antrea Windows Containerd ======"
+function build_and_deliver_antrea_windows_and_linux_docker_images {
+    echo "====== Cleanup Antrea Installation Before Delivering Antrea Windows and Antrea Linux Docker Images ====="
+    clean_antrea
+    kubectl delete -f ${WORKDIR}/antrea-windows.yml --ignore-not-found=true || true
+    kubectl delete -f ${WORKDIR}/kube-proxy-windows.yml --ignore-not-found=true || true
+    kubectl delete daemonset antrea-agent -n kube-system --ignore-not-found=true || true
+    kubectl delete -f ${WORKDIR}/antrea.yml --ignore-not-found=true || true
+
+    prepare_env
+    ${CLEAN_STALE_IMAGES}
+    ${PRINT_DOCKER_STATUS}
+    chmod -R g-w build/images/ovs
+    chmod -R g-w build/images/base
+  
+    if [[ "$TESTCASE" == "windows-networkpolicy-process" ]]; then
+        make windows-bin
+    fi
+
+    export_govc_env_var
+
+    # Enable verbose log for troubleshooting.
+    sed -i "s/--v=0/--v=4/g" build/yamls/antrea.yml build/yamls/antrea-windows.yml
+
+    if [[ "${PROXY_ALL}" == true ]]; then
+        echo "====== Updating yaml files to enable proxyAll ======"
+        KUBERNETES_SVC_EP_IP=$(kubectl get endpoints kubernetes -o jsonpath='{.subsets[0].addresses[0].ip}')
+        KUBERNETES_SVC_EP_PORT=$(kubectl get endpoints kubernetes -o jsonpath='{.subsets[0].ports[0].port}')
+        KUBERNETES_SVC_EP_ADDR="${KUBERNETES_SVC_EP_IP}:${KUBERNETES_SVC_EP_PORT}"
+        sed -i "s|.*kubeAPIServerOverride: \"\"|    kubeAPIServerOverride: \"${KUBERNETES_SVC_EP_ADDR}\"|g" build/yamls/antrea.yml build/yamls/antrea-windows.yml
+        sed -i "s|.*proxyAll: false|      proxyAll: true|g" build/yamls/antrea.yml build/yamls/antrea-windows.yml
+    fi
+
+    cp -f build/yamls/*.yml $WORKDIR
+    echo "====== Delivering Antrea to all Nodes ======"
+    set +e
+    deliver_antrea_windows &> deliver_antrea_windows.log &
+    deliver_antrea_windows_pid=$!
+    deliver_antrea_linux
+    linux_result=$?
+    wait $deliver_antrea_windows_pid
+    wait_result=$?
```

Comment:
maybe rename this to `windows_result` for readability

Reply:
updated